src/defs.h

```diff
@@ -206,7 +207,8 @@ enum BKZFlags {
   BKZ_MAX_TIME = 8,
   BKZ_BOUNDED_LLL = 0x10,
   BKZ_AUTO_ABORT = 0x20,
-  BKZ_DUMP_GSO = 0x40
+  BKZ_DUMP_GSO = 0x40,
+  BKZ_GH_BND = 0x80
```

Comment:
Would it make sense to add a command line option to main.ccp so it's easier to test?